### DISCUSSION TITLE
Field for FAQ

### DISCUSSION THREAD

**Discussion by koksikus** [directus/directus#14039] [Archive]
In my records and I would like to have place for a something like discussion, i.e. I will have a question and someone will answer me and I would like to keep it in my record. I know there are comments in the sidebar and that's great. But if I would like to make a broader statement, which type of field will be the best for that, Repeater, WYSIWG or Textarea? What do you think?

**Comment by rijkvanzanten** [comment]
You could also consider setting up a one to many interface to a collection with a question and answer field 🤔
